### ISSUE TITLE
Free space check after checking active jobs not working properly

### ISSUE THREAD

**Issue by gepard-me** [swar/Swar-Chia-Plot-Manager#734]
Free space check shows the same amount of free space despite 8 running jobs. It should deduct 109,000,000,000 for each job.

> 2021-05-22 09:33:43 [INFO]: Free space before checking active jobs: {'E:': 949150167040}
> 2021-05-22 09:33:43 [INFO]: Free space after checking active jobs: {'E:': 949150167040}

ver. 0.1.0b
Windows 10
Python 3.7.0

**Comment by stubmirror** [comment]
think i have the same issue in ubuntu, it keeps adding jobs for a directory not accounting for running jobs, until the disk is actually full, then it stops.

**Comment by Ron-ski** [comment]
Yes I encountered this overnight, I added another drive late last night and added it to the config, so some plots were already running prior to adding this drive, so they would not have taken the new drive into account of course.

But some new plots were also scheduled to the old drive after adding the new one, it seems it fails to take into account the space required by currently running plots.

Another factor is I have two PC's plotting, so obviously one doesn't know what the other is doing.

So I think it needs a two stage approach.

1. Take into account already running plots when deciding on free space
2. Check free space for already running plots that are approaching completion and if not enough space move a completed plot from one drive to another.

Number two is what I did this morning manually, across the two computers I had four stuck at 100% and another two almost finished plotting destined for the same drive. I simply moved 6 completed plots from one drive to another, the stuck plots then all gradually cleared.

Word of warning, when moving plots the Chia GUI will say you have X invalid plots and they should be deleted, they are not invalid if they are the ones just moved, I usually just restart the GUI.

The free space feature is still very useful, it saved the whole process from grinding to a halt, and just required a little manual intervention to clear the log jam.
